### ISSUE TITLE
Add an Item – Item Relations – Adding Content Lacks Status Message

### ISSUE THREAD

**Issue by aviannamiller** [omeka/plugin-ItemRelations#23]
"Description: On the Add Items page, under the Item Relations, it was found that adding content lacks a status message. Currently, when using the ‘Add a Relation’ button a new row is added to the table above, but there is no status message announcement of this new content on the page. When new content is added to the page behind the keyboard focus, there should be a status message, so users don’t have to navigate backwards up the page and check manually. A status message of ‘New row added to the table’ would let users know they have successfully added this using the ‘Add a relation’ button.  

Expected Results: When content is added to the page in a previously navigated section, there is a status message to inform users of new content added. 

Actual Results: When users add a row using the ‘Add a relation’ button there is no status message of successfully adding it"

1.3.1 Info and Relationships; Level A; WCAG 2.0
